crates/bevy_math/src/curves/mod.rs

```diff
@@ -0,0 +1,38 @@
+use thiserror::Error;
+
+mod fixed;
+mod variable;
+mod variable_linear;
+
+pub use fixed::*;
+pub use variable::*;
+pub use variable_linear::*;
+
+pub type CurveCursor = u16;
```

Comment:
As someone new to animation, I have *no* idea what this does on first glance, especially if all I had access to was the docs.

Reply:
Also, I'm not a fan of type aliases, I much prefer strong types.
So I would recommend
```rust
pub struct CurveCursor(u16);
```

Reply:
That make easy to change it to a `u32` for instance in case we need more keys, but I doubt

Reply:
I made some changed and renamed it to KeyframeIndex

Reply:
doc created